extensions/BMO/Extension.pm

```diff
@@ -713,6 +713,18 @@ sub bug_format_comment {
         }
     });
 
+    # link github pull requests and issues
+    push (@$regexes, {
+        match => qr#\s([A-Za-z0-9_\.-]+\/[A-Za-z0-9_\.-]+\#[0-9]+)\b#mx,
+        replace => sub {
+            my $args = shift;
+            my $owner = html_quote($args->{matches}->[0]);
+            my $repo = html_quote($args->{matches}->[1]);
+            my $number = html_quote($args->{matches}->[2]);
```

Comment:
I have a question. How does it know what is `[0]`, `[1]`, and `[2]`? It seems to me there are no such groups in the regexp. Should it be `([A-Za-z0-9_\.-]+)\/([A-Za-z0-9_\.-]+)\#([0-9]+)` instead? I'm not familiar with Perl, so I may be missing something.